src/widgets/refinement-list/refinement-list.js

```diff
@@ -308,7 +271,7 @@ export default function refinementList({
       escapeFacetValues,
       transformItems,
     });
-  } catch (e) {
+  } catch (error) {
     throw new Error(usage);
```

Comment:
It's not related to this PR but it's an occasion to ask the question.
Is there a specific policy regarding catching/displaying errors?

Here for example we choose to forget about the original error but it's also hiding some potentially helpful clues for the support team? Is this a choice?

Reply:
This is a pattern in InstantSearch specifically. Usually it _does_ get fixed by rereading the input expected, but probably we should put the original error in there too (like on AlgoliaError on the client)